common/win_logging.go

```diff
@@ -0,0 +1,12 @@
+// +build !linux,!darwin
```

Comment:
`// +build windows` makes a little more sense I guess, but I see nacl and plan9 omitted from linux build too (the positive case makes more sense than the negative case, for both of these files) but I guess this will just blow up on some other system anybody might try.